tests/app.rs

```diff
@@ -0,0 +1,35 @@
+use wasm_bindgen_test::{wasm_bindgen_test_configure, wasm_bindgen_test};
```

Comment:
I'll leave this in for now. Basically template boilerplate, demonstrating how to do things...